src/Components/LoginForm.js

```diff
@@ -37,23 +39,30 @@ export const  LoginForm = ({ toggleSignup }) =>
           email_id: email.current.value,
           password: password.current.value
       }
-      console.log(payload);
-      AuthenticationService.SignIn(payload).then((response)=>{
+      console.log("data",payload);
+      AuthenticationService.SignIn(payload).then((response) => {
         const message = response.data.Return;
-        console.log(message);
-        window.location.href = '/allprojects';
+        console.log("message", message);
+        const token = response.data.token;
+        // Store the token in local storage
+        localStorage.setItem('token', token);
+        console.log("token1", token);
+        // localStorage.getItem(token);
```

Comment:
Remove unused commented parts

Reply:
Sure, I will remove and commit the changes.